src/App.tsx

```diff
@@ -1,26 +1,183 @@
-/* eslint-disable max-len */
+/* eslint-disable react-hooks/rules-of-hooks */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { TodoList } from './components/TodoList';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [status, setStatus] = useState<TodoStatus>(TodoStatus.all);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [editingTodo, setEditingTodo] = useState(0);
```

Comment:
Consider changing this to a more meaningful name because when I read it the first time I couldn't understand why you need this state